src/Request.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace ChrisJP\TTS;
+
+/**
+ * The Request class handles the sending and receiving of API requests using cURL methods.
+ * 
+ */
+class Request
+{
+
+    /**
+     * The base URL to connect to
+     *
+     * @var string
+     */
+    private string $baseURL;
+
+    /**
+     * Number of seconds after which to timeout the request
+     * By default cURL requests never timeout. Here, the default is set to 30 seconds.
+     * This can be overridden via the constructor.
+     *
+     * @var integer
+     */
+    private int $timeout = 30;
+
+    /**
+     * Stores the output from curl_getinfo()
+     *
+     * @var array
+     */
+    private array $info;
+
+    /**
+     * Stores the output from curl_exec()
+     *
+     * @var string
+     */
+    private string $response;
+
+    public function __construct(string $baseURL, int $timeout = 30)
+    {
+        $this->baseURL = $baseURL;
+        $this->timeout = $timeout;
+    }
+
+    /**
+     * Set up a cURL session and execute the request
+     * If no parameters are passed to this a simple GET will be performed on the URL
+     *
+     * @param string $payload
+     * @param boolean $POST
+     * @param array|null $headers
+     * @param boolean $headerInOutput
+     * @return $this
+     */
+    public function sendRequest(array|string $payload = '', bool $POST = true, array|null $headers = [], bool $headerInOutput = false)
+    {
+        // The data being sent with this request needs to be a string
+        // If an array has been passed use http_build_query() to generate a URL-encoded query string
+        $payload = is_array($payload) ? http_build_query($payload) : $payload;
+
+        // Initialise cURL session
+        $ch = curl_init();
+
+        // Set base URL to send request to, appending our payload as a query string if this is a GET
+        $URL = $POST ? $this->baseURL : $this->baseURL . $payload;
```

Comment:
All-uppercase vars are usually reserved for constants, I would make these lowercase.